torchx/schedulers/__init__.py

```diff
@@ -22,6 +22,7 @@
     "gcp_batch": "torchx.schedulers.gcp_batch_scheduler",
     "ray": "torchx.schedulers.ray_scheduler",
     "lsf": "torchx.schedulers.lsf_scheduler",
+    "kueue_job": "torchx.schedulers.kueue_job_scheduler"
```

Comment:
can we use just `kueue` and `kueue_scheduler` to keep it inline with other schedulers? Seems like the project name is just Kueue rather than Kueue Job https://github.com/kubernetes-sigs/kueue